scripts/ci/distcheck.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+# CC LD AR CXX has to be predifubed
+#
+
+export PKG_CONFIG_PATH="$HOME/cunit-install/x86_64/lib/pkgconfig:${PKG_CONFIG_PATH}"
+
+CWD=$(dirname "$0")
+TDIR=`mktemp -d -p ~`
+
+cd ${TDIR}
```

Comment:
Why do you need a separate temp dir. Are these docker images isolated or not?

Reply:
I use it do debug build manually. I.e. I can run docker with /bin/bash and than type commands. For me it's more convenient to do it in some directory which I can delete and restart again.